src/node_code_cache.h

```diff
@@ -7,6 +7,8 @@
 
 namespace node {
 
+extern bool native_module_has_code_cache;
```

Comment:
Can this be `const`, to clarify that it’s not global mutable state?

Reply:
Even `constexpr`.
Actually it probably should move to .gyp... I need to keep this in mind

Reply:
@refack This is the one bit that we can generate without a chicken-and-egg problem...what what good does it do if we can't generate the actual static data along with it?

Reply:
Although, come to think of it, now that we refactored the native module compilation in C++, we may be able to solve that chicken-and-egg problem more easily...

Reply:
> @refack This is the one bit that we can generate without a chicken-and-egg problem...what what good does it do if we can't generate the actual static data along with it?

I mean, GYP knows if the code cache is a stub or real (if `node_code_cache_stub.cc` is included or `generated/node_code_cache.cc`) - So it can inject `#define HAS_REAL_CODE_CACHE 0 / 1` and we don't need this as code at all.

Reply:
P.S. we need to push for GCC7 and get `constexpr if` that way we can write code that doesn't compile decided by const code values.